dataverse-webapp/src/test/java/edu/harvard/iq/dataverse/harvest/client/HarvestingClientsServiceIT.java

```diff
@@ -97,6 +97,7 @@ public void shouldUpdateHarvestingClient() {
     }
 
     @Test
+    @Ignore
```

Comment:
I had to ignore this test as it broken on various levels and I don't see the way to fix it (I wonder if it was ever working).
The first problem is that the method being tested is @Asynchronous and it fires long after the asserts are called causing them to fail and also generating exception for trying to execute EJB method when app is not deployed.
On the other hand when we remove the @Asynchronous annotation, we end with problems with transactions (most probably we do not have access to harvesting client that was added).